ISSUE_TEMPLATE.md

```diff
@@ -34,3 +34,7 @@ Please include relevant error messages and/or screenshots.
 <!--
 If you can, link to the line of code that might be responsible for the problem.
 -->
+
+---
+
+JIRA tickets are created for all valid issues and then Help Wanted GitHub issues are opened up for those JIRA tickets that our developer teams are not immediately working on. The internal goal is to bring the desktop repo into our day-day process with the hope that this process change will mean more frequent desktop releases with faster feature development and turnaround time on bug fixes.
```

Comment:
Propose:

```
[JIRA tickets]() are created for all valid bug reports filed here. Bugs and features that the Mattermost core team are not immediately working on are opened as [Help Wanted GitHub issues]() and welcome to contributions. 

Our goal is to bring the Mattermost desktop repository into our day-to-day internal process with the hope of having more frequent desktop releases with faster feature development and turnaround time on bug fixes.

Reply:
@wget does this help? Anything you'd like to see here to help with the discussion we've been having in Desktop channel?

Reply:
@amyblais we can update to the above text and then I think we're good to merge

Reply:
NVM, already updated, thanks!